### DISCUSSION TITLE
Specflow Allure - : 'Unable to change the container context because the test context is active.'

### DISCUSSION THREAD

**Discussion by SAM-D-007** [allure-framework/allure2#2738] [Questions & Support] [answered]
I am trying to setup Alure report in my Specflow project to integrate AllureReports Tab in Azure devops pipeline but i am getting the exception System.InvalidOperationException: 'Unable to change the container context because the test context is active.' while running in my local.

Specflow C#
.NET Version : .NET CORE 7.0
UnitTestFramework:  XUNIT

Installed packages screenshot.
![Allure](https://github.com/user-attachments/assets/b4782252-7364-4c0c-933c-32c2c980103b)

Can some please help me with this

Thanks,
Shyam

**Comment by delatrie** [comment, marked as answer]
Hi, @SAM-D-007 !
Unfortunately, Allure.SpecFlow doesn't support xUnit. Consider moving to Reqnroll because SpecFlow is not maintained anymore. Renroll is maintained by the original creator of SpecFlow and Allure.Reqnroll supports all three major runners: NUnit, xUnit, and MSTest.

**Reply by SAM-D-007** [reply]
Thank you for the response @delatrie ,  SpecFlow+ LivingDoc Azure equivalent extension to publish results  is a prerequisite for me to migrate from Specflow to Reqnroll which is not yet supported by Reqnroll. 
Can we expect Allure.SpecFlow support for Xunit in near future, otherwise i would consider to move to MSTest Unit test framework.
